packages/frontend/apps/electron/renderer/app.tsx

```diff
@@ -111,6 +117,49 @@ window.addEventListener('focus', () => {
 });
 frameworkProvider.get(LifecycleService).applicationStart();
 
+events?.applicationMenu.openAboutPageInSettingModal(() =>
```

Comment:
The two events of applicationMenu are moved out of `@affine/core`. If we want to add more in the future, we can put them in a separate file.